packages/manager/tools/component-rollup-config/LICENSE

```diff
@@ -13,14 +13,14 @@ modification, are permitted provided that the following conditions are met:
   this list of conditions and the following disclaimer in the documentation
```

Comment:
This file mustn't be moved from `packages/manager/apps/pci/`.

Please prefer add a new `LICENSE` file to the new module.